### ISSUE TITLE
about the human 3.6m

### ISSUE THREAD

**Issue by lvandoit** [CCChenfei/for_human3.6M#1]
hi：
   Sorry to bother you, I am a graduate student who mainly studies human pose estimation, but I have a problem now, I don't have human 3.6m dataset,So,do you have it?thanks for your any reply. @CCChenfei

**Comment by CCChenfei** [comment]
Hi，
     Yes I have. But the dataset is about 100G，how can I give it to you?

发自我的 iPhone

在 2019年12月1日，下午2:18，lvandoit <notifications@github.com<mailto:notifications@github.com>> 写道：


hi：
Sorry to bother you, I am a graduate student who mainly studies human pose estimation, but I have a problem now, I don't have human 3.6m dataset,So,do you have it?thanks for your any reply. @CCChenfei<https://github.com/CCChenfei>

**Comment by lvandoit** [comment]
i am so happy to get your reply,Are you in China now?&nbsp; can you &nbsp;share it online or will i&nbsp;send you a hard drive?i am doing the research about the human pose estimation.
  

 

 ------------------&nbsp;原始邮件&nbsp;------------------
  发件人:&nbsp;"Fei Chen"<notifications@github.com&gt;;
 发送时间:&nbsp;2019年12月1日(星期天) 下午2:22
 收件人:&nbsp;"CCChenfei/for_human3.6M"<for_human3.6M@noreply.github.com&gt;;
 抄送:&nbsp;"2324524489"<2324524489@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
 主题:&nbsp;Re: [CCChenfei/for_human3.6M] about the human 3.6m (#1)

 

Hi， 
Yes I have. But the dataset is about 100G，how can I give it to you? 

发自我的 iPhone 

在 2019年12月1日，下午2:18，lvandoit <notifications@github.com<mailto:notifications@github.com&gt;&gt; 写道： 


hi： 
Sorry to bother you, I am a graduate student who mainly studies human pose estimation, but I have a problem now, I don't have human 3.6m dataset,So,do you have it?thanks for your any reply. @CCChenfei<https://github.com/CCChenfei&gt;

**Comment by CCChenfei** [comment]
Yes I am in China. Later I will try to put it online. But  today I am not in school. I may reply to you tomorrow.

发自我的 iPhone

在 2019年12月1日，下午2:29，lvandoit <notifications@github.com<mailto:notifications@github.com>> 写道：

i am so happy to get your reply,Are you in China now?&nbsp; can you &nbsp;share it online or will i&nbsp;send you a hard drive?i am doing the research about the human pose estimation.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Fei Chen"<notifications@github.com<mailto:notifications@github.com>&gt;;
发送时间:&nbsp;2019年12月1日(星期天) 下午2:22
收件人:&nbsp;"CCChenfei/for_human3.6M"<for_human3.6M@noreply.github.com<mailto:for_human3.6M@noreply.github.com>&gt;;
抄送:&nbsp;"2324524489"<2324524489@qq.com<mailto:2324524489@qq.com>&gt;;"Author"<author@noreply.github.com<mailto:author@noreply.github.com>&gt;;
主题:&nbsp;Re: [CCChenfei/for_human3.6M] about the human 3.6m (#1)



Hi，
Yes I have. But the dataset is about 100G，how can I give it to you?

发自我的 iPhone

在 2019年12月1日，下午2:18，lvandoit <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com&gt;&gt; 写道：


hi：
Sorry to bother you, I am a graduate student who mainly studies human pose estimation, but I have a problem now, I don't have human 3.6m dataset,So,do you have it?thanks for your any reply. @CCChenfei<https://github.com/CCChenfei&gt;

**Comment by lvandoit** [comment]
先生：
 &nbsp;&nbsp;&nbsp; 好的，非常感谢您的帮助，那我明天再联系您。祝好！
  

 

 ------------------&nbsp;原始邮件&nbsp;------------------
  发件人:&nbsp;"Fei Chen"<notifications@github.com&gt;;
 发送时间:&nbsp;2019年12月1日(星期天) 下午2:36
 收件人:&nbsp;"CCChenfei/for_human3.6M"<for_human3.6M@noreply.github.com&gt;;
 抄送:&nbsp;"2324524489"<2324524489@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
 主题:&nbsp;Re: [CCChenfei/for_human3.6M] about the human 3.6m (#1)

 

Yes I am in China. Later I will try to put it online. But today I am not in school. I may reply to you tomorrow. 

发自我的 iPhone 

在 2019年12月1日，下午2:29，lvandoit <notifications@github.com<mailto:notifications@github.com&gt;&gt; 写道： 

i am so happy to get your reply,Are you in China now?&amp;nbsp; can you &amp;nbsp;share it online or will i&amp;nbsp;send you a hard drive?i am doing the research about the human pose estimation. 




------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
发件人:&amp;nbsp;"Fei Chen"<notifications@github.com<mailto:notifications@github.com&gt;&amp;gt;; 
发送时间:&amp;nbsp;2019年12月1日(星期天) 下午2:22 
收件人:&amp;nbsp;"CCChenfei/for_human3.6M"<for_human3.6M@noreply.github.com<mailto:for_human3.6M@noreply.github.com&gt;&amp;gt;; 
抄送:&amp;nbsp;"2324524489"<2324524489@qq.com<mailto:2324524489@qq.com&gt;&amp;gt;;"Author"<author@noreply.github.com<mailto:author@noreply.github.com&gt;&amp;gt;; 
主题:&amp;nbsp;Re: [CCChenfei/for_human3.6M] about the human 3.6m (#1) 



Hi， 
Yes I have. But the dataset is about 100G，how can I give it to you? 

发自我的 iPhone 

在 2019年12月1日，下午2:18，lvandoit <notifications@github.com<mailto:notifications@github.com&gt;<mailto:notifications@github.com&amp;gt;&amp;gt; 写道： 


hi： 
Sorry to bother you, I am a graduate student who mainly studies human pose estimation, but I have a problem now, I don't have human 3.6m dataset,So,do you have it?thanks for your any reply. @CCChenfei<https://github.com/CCChenfei&amp;gt;

**Comment by lvandoit** [comment]
hi: 
   Sorry to bother you,It is convenient for you to share  the dataset for me?Because i need the dataset to do my experiments.please! @CCChenfei

**Comment by lvandoit** [comment]
hi:
Sorry to bother you, is it  convenient for you to share the dataset for me?Because i need the dataset to do my experiments.please! @CCChenfei

**Comment by CCChenfei** [comment]
@lvandoit I put it on 百度网盘，链接: https://pan.baidu.com/s/1UuPni7tLOL3QMyEFXJBJOA 提取码: infc

**Comment by lvandoit** [comment]
hi:
 &nbsp;&nbsp; It means a lot for me,Thank you very much
  

 

 ------------------&nbsp;原始邮件&nbsp;------------------
  发件人:&nbsp;"notifications"<notifications@github.com&gt;;
 发送时间:&nbsp;2019年12月3日(星期二) 中午12:38
 收件人:&nbsp;"CCChenfei/for_human3.6M"<for_human3.6M@noreply.github.com&gt;;
 抄送:&nbsp;"2324524489"<2324524489@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
 主题:&nbsp;Re: [CCChenfei/for_human3.6M] about the human 3.6m (#1)

 

 
@lvandoit I put it on 百度网盘，链接: https://pan.baidu.com/s/1UuPni7tLOL3QMyEFXJBJOA 提取码: infc

**Comment by renguoyin123** [comment]
Your link is bad, can you send a  Useful links for HUMAN3.6M in 267G
